CondFormats/SerializationHelper/test/test_catch2_SerializationHelper.cc

```diff
@@ -0,0 +1,147 @@
+// -*- C++ -*-
+//
+// Package:     CondFormats/SerializationHelper
+// Class  :     test_catch2_SerializationHelper
+//
+// Implementation:
+//     [Notes on implementation]
+//
+// Original Author:  Christopher Jones
+//         Created:  Wed, 31 May 2023 19:12:55 GMT
+//
+
+// system include files
+#include "catch.hpp"
+
+// user include files
+#include "CondFormats/SerializationHelper/interface/SerializationHelper.h"
+
+namespace {
+  struct Test {
+    Test() = default;
+    Test(float iA, int iB) : a_{iA}, b_{iB} {}
+
+    float a_;
+    int b_;
+
+    template <class Archive>
+    void serialize(Archive& ar, const unsigned int version) {
+      ar& a_;
```

Comment:
What exactly happens here? Something along `Archive::operator&(&ar, a_)` perhaps? But what does that end up doing? (I guess I'm mostly confused of the use of `operator&()` for serialization)

Reply:
This all comes from the boost::serialization API. I just did a 'cut-n-paste' of what the conditions code generator was doing. The unit test showed it did work.